### PR TITLE
GraphQL: Fix formatting for directives in fields

### LINKED ISSUE #9068: GraphQL directives are not indented
**Prettier 2.1.1**
[Playground link](https://prettier.io/playground/#N4Igxg9gdgLgprEAuEMCeAHOACAigVzgCc1tgAdKbbAZwgFs4Di0AKASwBMlsBJAEQCEAGloM4AeRgALYvwCGMeTwDKMIuygBzQQEpV6zTuwABTnAxE4YRXE6n2NAIL4ZCGOxvxOlagDdiGnZoGh4ANUDgqBpBSgBfEGEQCAwPEORQeSIiCAB3AAUshBpkEHkAG1z5NBKkgCMieTAAazgYFQwmo2R1QiS4ejq7c04AGXltfHktOAAxCCJ6RQ9tUvlXCESQaRh6coB1aXZ4Gk6wOBVi4-Y-Y7RSsBpakE0aYhh8xq0l5AAzCreSQAVjQAB4AIUaLTaKnkjFGmjgfwBcGBYJURnKTHwEHgyPKgJAnSIbyIpS0jQw0gAjuUtpZNDB9lwZMgABwABiSlggb32lNKljgpICW2pOPgnxSJRQ8hoAFooHA7HYtlZxewrJ9pj8kP8CaiQG96OwekQ+kbMdjcUi9Sikko6szOKykAAmB2NdjlIwAYQYupAwoArFt8G8ACryOoy-WEvyEXhQcywFRgDSpJzJtRoLH4t5xOJAA)
```sh
--parser graphql
```

**Input:**
```graphql
type Query {
  someQuery(id: ID!, someOtherData: String!): String! @deprecated @isAuthenticated
  versions: Versions!
}
```

**Output:**
```graphql
type Query {
  someQuery(id: ID!, someOtherData: String!): String!
  @deprecated
  @isAuthenticated
  versions: Versions!
}

```

**Expected behavior:**
Output from `prettier@2.0.5`:

```graphql
type Query {
  someQuery(id: ID!, someOtherData: String!): String!
    @deprecated
    @isAuthenticated
  versions: Versions!
}
```

This changed in #7721. 

IMO, this is a regression to readability. Especially since directives are applied to the thing before rather than after it, unlike decorators in JS or annotations in Java. You could argue it's hard to grok regardless, and I would agree with you! 😀 But indenting in general is usually related to what's above it and not below it, so I do think the indentation helps readability in this case.

### PR DESCRIPTION
**Pull request by sosukesuzuki**
Fixes #9068

- [x] I’ve added tests to confirm my change works.
- [ ] (If changing the API or CLI) I’ve documented the changes I’ve made (in the `docs/` directory)
- [x] (If the change is user-facing) I’ve added my changes to `changelog_unreleased/*/pr-XXXX.md` file following `changelog_unreleased/TEMPLATE.md`.
- [x] I’ve read the [contributing guidelines](https://github.com/prettier/prettier/blob/master/CONTRIBUTING.md).

**✨[Try the playground for this PR](https://prettier.io/playground-redirect)✨**

### CODE REVIEW

File: src/language-graphql/printer-graphql.js [hunk 1]
```diff
@@ -611,7 +611,24 @@ function printDirectives(path, print, n) {
...
+  }
+
+  return concat([
+    " ",
+    group(
+      indent(
+        concat([
+          softline,
+          join(
+            concat([ifBreak("", " "), softline]),
```
Comment (thorn0) [inline src/language-graphql/printer-graphql.js:625]:
```suggestion
            line,
```

**Comment by SimenB** [comment]
> /cc @SimenB Sorry I'm not familiar with GraphQL, could you check the behavior?

Works great for me, thanks! 👍

File: src/language-graphql/printer-graphql.js (hunk 1 above)
Comment (sosukesuzuki) [reply src/language-graphql/printer-graphql.js:625]: Probably no.

**Review by thorn0 — APPROVED** [review]

**Review by evilebottnawi — APPROVED** [review]

**Pull request merged**